WooCommerce/Classes/Authentication/AuthenticationManager.swift

```diff
@@ -289,6 +289,22 @@ extension AuthenticationManager: WordPressAuthenticatorDelegate {
             return navigationController.show(mismatchAccountUI, sender: nil)
         }
 
+        if let matchedSite = matcher.matchedSite(originalURL: siteURL),
+           matchedSite.isWooCommerceActive == false {
+            let viewModel = NoWooErrorViewModel(
+                siteURL: siteURL,
+                showsConnectedStores: matcher.hasConnectedStores,
+                showsInstallButton: matchedSite.isJetpackConnected,
+                onSetupCompletion: { [weak self] siteID in
+                guard let self = self else { return }
+                self.storePickerCoordinator = StorePickerCoordinator(navigationController, config: .login)
+                self.storePickerCoordinator?.onDismiss = onDismiss
+                self.storePickerCoordinator?.didSelectStore(with: siteID, onCompletion: onDismiss)
```

Comment:
Super Nit: 
Could we create a private method for the `StorePickerCoordinator` related code and call it here? 

I am hoping that we can create a private method [with this code](https://github.com/woocommerce/woocommerce-ios/blob/0728bda72160fec673d06ebd0ac250824d338d2d/WooCommerce/Classes/Authentication/AuthenticationManager.swift#L308-L314) and reuse it in both places. Please correct me if I am wrong.

Reply:
Good idea! Updated in f797fc567802e8116134fcf6ece8f77d4f16d9f0 to reuse this piece of code since we need it in 3 places now.